vernac/comFixpoint.mli

```diff
@@ -18,13 +18,21 @@ open Vernacexpr
 
 (** Entry points for the vernacular commands Fixpoint and CoFixpoint *)
 
+val do_fixpoint_interactive :
+  (* When [false], assume guarded. *)
```

Comment:
This comment is leftover from when typing flags were passed separately from the environment, it should just be removed.

Reply:
I coul'd not get what it was about so I did not touch it. I can remove it.

Reply:
It's from e0547f9e9134a9fff122df900942a094c53535c3 to be precise

Reply:
removed